crates/napi/src/minify.rs

```diff
@@ -118,12 +90,12 @@ pub fn minify(
 
 #[napi]
 pub fn minify_sync(input: Buffer, opts: Buffer) -> napi::Result<TransformOutput> {
-    let code: MinifyTarget = serde_json::from_slice(&input)?;
+    let code = String::from_utf8_lossy(&input).into_owned();
```

Comment:
why lossy?
it looks like the other side is using `Buffer.from` which itself will do a lossy conversion to utf8 so here we should be able to assert proper utf8